src/modal.ts

```diff
@@ -56,12 +56,12 @@ globalStylesheet.insertRule(`
 @customElement('glide-core-modal')
 @final
 export default class Modal extends LitElement {
-  /* c8 ignore start */
+  /* v8 ignore start */
   static override shadowRootOptions: ShadowRootInit = {
     ...LitElement.shadowRootOptions,
     mode: window.navigator.webdriver ? 'open' : 'closed',
   };
-  /* c8 ignore end */
+  /* v8 ignore stop */
```

Comment:
I'm planning to audit these comments after we've finished migrating to Playwright. But they're good to update as we go, where we remember to, in case new features are added. That way CI fails if the new feature isn't covered.